test/E2ETest/Microsoft.Test.E2E.AspNet.OData/BulkOperation/BulkOperationController.cs

```diff
@@ -249,15 +249,17 @@ public ITestActionResult PatchEmployees([FromBody] DeltaSet<Employee> coll)
 
         [ODataRoute("Employees")]
         [HttpPost]
+        [EnableQuery]
         public ITestActionResult Post([FromBody] Employee employee)
         {
-            InitEmployees();
+            //InitEmployees();
```

Comment:
is this meant to be added in the next PR?

Reply:
If the code is not relevant to this PR, we should delete it instead of commenting it out

Reply:
I would prefer creating a new test instead of modifying the existing one, since return `Ok()` from a `Post()` request is still supported.

Reply:
Deleted the code.
@habbes This controller method is for bulk ops and we don't need to have a test that returns `Ok()`